docker/core-site.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+
+<configuration>
```

Comment:
How is this file used?

Reply:
It's used in the spark-base image to configure the Google hadoop connector.